code/modules/medical/medical_effect.dm

```diff
@@ -0,0 +1,23 @@
+/datum/medical_effect
```

Comment:
i hand-wrote these because disease code is crap and I didn't want it interfering